client/src/components/AlumniDirectory.js

```diff
@@ -79,9 +79,13 @@ export default class AlumniDirectory extends Component {
 
     async componentWillMount() {
         let result = await this.getEntries()
+        let totalPages = 0;
+        if (result.alumni !== null) {
+            totalPages = Math.ceil(result.alumni.length/pageSize);
+        }
         this.setState({
             entries: result.alumni,
-            totalPages: Math.ceil(result.alumni.length/pageSize),
+            totalPages: totalPages,
             numEntries: result.alumni.length
```

Comment:
This fixed a weird bug. I'm still not entirely sure what caused it, but if there were no alumni it would throw an error

Reply:
An alternative way to do this in-line would be using javascript's short-circuit evaluation like so`result.alumni && Math.ceil(result.alumni.length/pageSize)`